graphkb/match.py

```diff
@@ -305,8 +305,8 @@ def compare_positional_variants(
     ):
         return False
 
-    if 'break2Start' in variant:
-        if 'break2Start' not in reference_variant:
+    if variant.get('break2Start'):
+        if not reference_variant.get('break2Start'):
```

Comment:
Should these be .get('break2Start', None), to also make the 'break2Start' property optional?

Reply:
no need, None is the default second argument to get. https://docs.python.org/3/library/stdtypes.html#dict.get